### PR TITLE
ESA-65

### PR DESCRIPTION
**Pull request by vmartos**
- Migration of Dashboard Canvas objects to VAS
- Override AddDefaultItem to match Longomatch defaults 
- Subscribe to create delete events instead of IanalsysWindows
- Create Menu in Longomatch timeline and moved some menus

### CODE REVIEW

File: LongoMatch.Core/Store/ActionLink.cs [hunk 1]
```diff
@@ -35,11 +35,11 @@ namespace LongoMatch.Core.Store
 	[PropertyChanged.ImplementPropertyChanged]
 	public class ActionLinkLongoMatch : ActionLink
 	{
-		public ActionLinkLongoMatch ()
+		public ActionLinkLongoMatch () : base ()
```
Comment (ylatuya) [inline LongoMatch.Core/Store/ActionLink.cs]: The base default constructor is executed automatically, see https://msdn.microsoft.com/en-us/library/aa645606%28v=vs.71%29.aspx

File: LongoMatch.GUI/Gui/Menu/SportsPlaysMenu.cs [hunk 2]
```diff
@@ -226,14 +226,6 @@ void CreateMenu ()
 			snapshot.Activated += (sender, e) => Config.EventsBroker.EmitSnapshotSeries (plays.FirstOrDefault ());
 			Add (snapshot);
 
-			duplicate = new MenuItem ("");
-			duplicate.Activated += (sender, e) => Config.EventsBroker.EmitDuplicateEvent (plays);
-			Add (duplicate);
-
-			del = new MenuItem ("");
```
Comment (ylatuya) [inline LongoMatch.GUI/Gui/Menu/SportsPlaysMenu.cs]: Is "Delete" removed because it's in vas now?

File: LongoMatch.Services/EventsManager.cs [hunk 3]
```diff
@@ -61,7 +61,7 @@ void HandlePlayerSubstitutionEvent (SportsTeam team, PlayerLongoMatch p1, Player
 
 				try {
 					evt = ((ProjectLongoMatch)openedProject).SubsitutePlayer (team, p1, p2, reason, time);
-					analysisWindow.AddPlay (evt);
```
Comment (ylatuya) [inline LongoMatch.Services/EventsManager.cs]: I would remove AddPlay from IAnalysisWindow too.

**Comment by ylatuya** [comment]
Make sure you add the new type mappings if needed.

File: LongoMatch.GUI/Gui/Menu/SportsPlaysMenu.cs (hunk 2 above)
Comment (vmartos) [inline LongoMatch.GUI/Gui/Menu/SportsPlaysMenu.cs]: Yes! Delete and NewPlay menus has been moved to PlaysMenu in VAS

File: LongoMatch.Services/EventsManager.cs (hunk 3 above)
Comment (vmartos) [inline LongoMatch.Services/EventsManager.cs]: AddPlay and DeletePlays has been removed in IAnalisysWindowBase in vas:
https://github.com/fluendo/VAS/pull/17/files#diff-2e47bd5b217c112bc5b14e93abe6cf3cL34

File: LongoMatch.Core/Store/ActionLink.cs (hunk 1 above)
Comment (vmartos) [inline LongoMatch.Core/Store/ActionLink.cs]: mmm, Seems that ActionLink is entirely migrated to VAS, and with a rebase i didn't git rm ActionLinkLongomatch. I'll remove this class from the Longomatch repo

**Comment by ylatuya** [comment]
👍  Just make sure unit tests passes after rebasing

**Pull request merged**